R/driver-sql-server.R

```diff
@@ -107,6 +107,30 @@ setMethod("dbExistsTable", c("Microsoft SQL Server", "SQL"),
   }
 )
 
+#' @details Call catalog-specific sp_tables to make sure we list the schemas in the
+#' appropriate database/catalog.
```

Comment:
```suggestion
#' @description
#' ## `odbcConnectionSchemas` 
#' Call catalog-specific `sp_tables` to make sure we list the schemas in the
#' appropriate database/catalog.
```

I've switched to this pattern because I think it's a bit clearer in the rendered docs, e.g. https://r-dbi.github.io/odbc/reference/SQLServer.html